frontend/public/components/project-overview.jsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { ListView, ListViewItem } from 'patternfly-react';
+
+import { ResourceLink } from './utils';
+import { modelFor } from '../module/k8s';
+
+export const ComponentLabel = ({text}) => <div className="co-component-label">{text}</div>;
+
+const ProjectOverviewListItem = ({obj}) => {
+  const {currentController, kind, metadata} = obj;
+  const {namespace, name, uid} = metadata;
+  const kindObj = modelFor(kind);
+  const heading = <h3 className="project-overview__item-heading">
+    <ResourceLink
+      kind={kind}
+      name={name}
+      namespace={namespace}
+    />
+  </h3>;
+
+  const additionalInfo = <div key={uid} className="project-overview__additional-info">
+    { currentController &&
+      <div className="project-overview__detail project-overview__detail--controller">
+        <ComponentLabel text={_.startCase(currentController.kind)} />
```

Comment:
I wonder if we shouldn't drop the labels in these rows. They seem fairly self-evident.

Reply:
Follow on?